Sources/Purchasing/OfferingsManager.swift

```diff
@@ -240,6 +240,7 @@ extension OfferingsManager.Error: CustomNSError {
 
     fileprivate var underlyingError: Error? {
         switch self {
+        case let .backendError(.networkError(error)): return error
```

Comment:
Should we use the `underlyingError` of the `BackendError` here?

Reply:
That's not defined, but it could be another way to handle this yeah.
I'd say for now we don't need to expose that for anything else so this is a bit more direct, but we could change it in the future.